### ISSUE TITLE
screenreader not announce number of votes near each variant in any poll.

### ISSUE THREAD

**Issue by AlexandrKozlovskiy** [UnigramDev/Unigram#2339]
In any poll screenreader (at least nvda) not announce percent of votes and number of votes near each variant. To know this information,we should read this,using object navigation of nvda,but it can be done more easier,when screenreader will be announce not only name of variant,but percent and number of votes.
